src/components/FilterListInput.tsx

```diff
@@ -36,7 +39,7 @@ export function FilterListInput({
 					aria-label="Filter items in the list"
 					placeholder="Search items..."
 				/>
-				<Button onClick={handleClear}>x</Button>
+				<Button onClick={handleClear}>Clear</Button>
```

Comment:
I like the change to clear, with bootstrap the x that was in the input didn't show any more and the button to clear was the best workaround but the x was awkard so it is much clearer (lol) with clear.